src/invoice/signature.rs

```diff
@@ -370,7 +370,11 @@ pub trait SecretKeyStorage {
     /// If no key is found, this will return a None.
     /// In general, if multiple keys match, the implementation chooses the "best fit"
     /// and returns that key.
-    fn get_first_matching(&self, role: &SignatureRole) -> Option<&SecretKeyEntry>;
+    fn get_first_matching(
```

Comment:
@thomastaylor312 Just thought it would be better to just have one method in the trait rather than having 2 methods that mostly does the same